.github/workflows/ci-build.yml

```
@@ -0,0 +1,48 @@
+
+name: CRDA CI Build
```

Please rename it as CI
```suggestion
name: CI
```

Also rename the file to CI.yaml